src/io/invoca/Docker.groovy

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/groovy
+package io.invoca;
+
+/* args
+    build_args  list (optional)
+    dockerfile  string
+    image_name  string
+*/
+
+@NonCPS
+def buildCommand(Map args) {
+    def String build_args = ["", *args.build_args].join(' --build-arg ')
+    def String cmd
+    
+    // http://label-schema.org/rc1/
+    cmd = "docker build -t ${args.image_name}:${env.GIT_COMMIT} \
```

Comment:
nit: I like the ability (especially for environment variables) to interpolate by just doing 
```
docker build -t ${args.image_name}:$GIT_COMMIT \
```
I think it reads a little easier

Reply:
Yeah, I kept going back and forth on this. Do you think for string interpolation we could do `$VARIABLE` and for directly variable usage do `env.`?

I think for the later scenario, `env.` reads better because it's clearer where it's coming from.

Reply:
Sounds good to me

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/6902de19-6a53-47cb-8d8b-a903a165a2ee-993a129b-4d12-4c8b-a2ad-03e265f1b37c-v1.png?transparent=1&palette=1&width=246)